.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - 7.10.0
+  - 8
+  - node
```

Comment:
For now, yes, but not in the future. Alternative is to put "8.0" and "node", or just "8.0" (but I like auto testing on new Node.js versions).